ouroboros-consensus/src/Ouroboros/Consensus/HardFork/Simple.hs

```diff
@@ -13,6 +13,10 @@ import           Cardano.Slotting.Slot (EpochNo)
 data TriggerHardFork =
     -- | Trigger the transition when the on-chain protocol major version (from
     -- the ledger state) reaches this number.
+    --
+    -- Note: Consensus does not enforce protocol versions to be successive.
+    -- That is, the ledger state is not restricted to transition from version
+    -- @x@ to @x+1@.
```

Comment:
I pinged @nfrisby.

Reply:
Seems like a reasonable place to me 👍

Reply:
Err. I'd rephrase it a bit though.

The ledger rules determine how the state is allowed to change (and the Shelley ledger eg might require it can only increase by one, contradicting the current phrasing of your comment). But Consensus makes no assumptions about it -- I suppose we do assume that it never goes down...

I think the more precise comment is that the HFC logic does not require that the trigger version for one era is the successor of the trigger version for the previous era.